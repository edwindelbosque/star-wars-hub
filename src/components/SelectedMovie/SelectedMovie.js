import React from 'react';
import QuoteContainer from '../QuoteContainer/QuoteContainer';
import CharactersContainer from '../CharactersContainer/CharactersContainer';


const SelectedMovie = ({ movie, characters }) => {
    return (
        <main className='selected-movie'>
            <QuoteContainer movie={movie} />
            <CharactersContainer characters={characters} />
        </main>
    )
}

export default SelectedMovie;