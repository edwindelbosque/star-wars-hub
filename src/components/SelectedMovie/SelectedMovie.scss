.selected-movie {
    display: flex;
}

.loading-image {
    height: 100px;
}