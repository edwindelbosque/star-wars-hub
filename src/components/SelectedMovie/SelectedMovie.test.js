import React from 'react';
import { shallow } from 'enzyme';
import SelectedMovie from './SelectedMovie';

describe('SelectedMovie', () => {
  it('component should match the snapshot', () => {
    const wrapper = shallow(
      <SelectedMovie
        characters={[]}
        movie={{ object: 'movieInfo' }}
      />
    )
    expect(wrapper).toMatchSnapshot();
  });
})