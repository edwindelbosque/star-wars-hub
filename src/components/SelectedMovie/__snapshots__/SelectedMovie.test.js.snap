// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedMovie component should match the snapshot 1`] = `
<main
  className="selected-movie"
>
  <QuoteContainer
    movie={
      Object {
        "object": "movieInfo",
      }
    }
  />
  <CharactersContainer
    characters={Array []}
  />
</main>
`;
